### ISSUE TITLE
Cannot deposit through a collection when there is only one work type

### ISSUE THREAD

**Issue by tworrall** [samvera/hyrax#3111]
### Descriptive summary

The "Deposit new work through this collection" feature does not work in Hyrax 2.1.0 **when there is only one work type defined**. When adding a file, it appears to be uploaded successfully but the "Add files" requirement does not get checked, and as a result the Save button is never enabled. 

### Expected behavior

Should be able to add a work through a collection when there is only one work type defined.

### Actual behavior

Can only able a work through a collection when there are multiple work types defined.

### Steps to reproduce the behavior

1. Go to a collection.
2. Click the "Deposit new work through this collection" button. The add new work form appears with the work type defaulted to the lone type.
3. Add metadata to describe the work. The "Describe your work" requirement gets a check mark.
4. Go to the Files tab and add a file. The "Add files" requirement will still have a red exclamation mark rather than a green check mark, and the Save button remains disabled even after checking the deposit agreement.

**Comment by vantuyls** [comment]
@no-reply slate this for 2.1.1?

**Comment by no-reply** [comment]
I'm not able to reproduce on Hyrax 2.1.0 at demo.curationexperts.com.

**Comment by vantuyls** [comment]
@tworrall @chrisdaaz can either of you replicate this problem and, if so, describe how?

**Comment by elrayle** [comment]
The key was that there was only a single work type.  When a second work type was added, the problem went away.  That said, I modified the engine cart generation process such that it generated only the GenericWork and I still want not able to reproduce the problem.

I will do a bit more exploration this afternoon to determine if there is some unusual setup that caused the problem before closing the issue.
